### PR TITLE
Enable interfaces for addtional external networks on the network nodes

### PR DESCRIPTION
**Pull request by rhafer**

### CODE REVIEW

File: crowbar_framework/app/models/neutron_service.rb [hunk 1]
```diff
@@ -277,6 +277,9 @@ def apply_role_pre_chef_call(old_role, role, all_nodes)
 
     network_nodes.each do |n|
       net_svc.allocate_ip "default", "public", "host",n
+      role.default_attributes["neutron"]["additional_external_networks"].each do |extnet|
+        net_svc.enable_interface "default", extnet, n
```
Comment (vuntz) [inline crowbar_framework/app/models/neutron_service.rb:303]: Hrm... Should we add a validation check that each network listed in `additional_external_networks` actually is defined in the network barclamp?

(And I guess we want to exclude some networks, like admin, public, os_sdn, etc.)

Comment (rhafer) [inline crowbar_framework/app/models/neutron_service.rb:303]: True. I guess we should do that in `validate_proposal_after_save` though.

Comment (rhafer) [inline crowbar_framework/app/models/neutron_service.rb:303]: I added a couple of validations

Comment (vuntz) [inline crowbar_framework/app/models/neutron_service.rb:303]: Awesome, thanks!

**Pull request merged**